Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/core/impl/IndexService.java

```diff
@@ -53,10 +53,26 @@
 		if (indexerRunningFlag.get()) {
 			throw new IndexerRunningException(indexerRunning);
 		}
-
-		return luceneUtility.queryIndex(fieldMap, operator);
+        // Fetches all the pages
+        return luceneUtility.queryIndex(fieldMap, operator, Integer.MIN_VALUE, Integer.MIN_VALUE);
```

Comment:
why don't you put in 0 for start index and -1 for page number? and then have logic in your lucene utility that checks for -1? This seems odd and not very intuitive.